Jenkinsfile

```diff
@@ -176,8 +179,8 @@ pipeline {
             }
             steps {
                 sh """
-                    $SBT clean compile package
-                    $SBT clean compile assembly
+                    sbt clean compile package
+                    sbt clean compile assembly
```

Comment:
Is this duplication necessary?

Reply:
Yes you're right, although neither might be needed. This depends on whether we wish to archive the .jar or packageBin zip (the latter of which is only needed for deployment/ redeployment) to Artifactory. However I believe @awharris did want the assembly jar to be archived - may need to reopen this dicussion.